terracumber/terraformer.py

```diff
@@ -46,6 +46,7 @@ def inject_repos(self, custom_repositories_json):
         """Set additional repositories into the main.tf, so they are injected by sumaform"""
```

Comment:
Since you are adding return values, explain them here.

Reply:
ok

Reply:
done